main.go

```diff
@@ -87,6 +87,7 @@ func main() {
 	fs.StringVar(&cli.MistHost, "mist-host", "127.0.0.1", "Hostname of the Mist server")
 	fs.StringVar(&cli.MistUser, "mist-user", "", "username of MistServer")
 	fs.StringVar(&cli.MistPassword, "mist-password", "", "password of MistServer")
+	fs.StringVar(&cli.MistPrometheus, "mist-prometheus", "", "Mist path for the prometheus metrics endpoint")
```

Comment:
You don't actually use `cli.MistPrometheus` to define the path. I think it's fine to have hardcoded `/mistmetrics` path. Probably no need to ever change it.

Reply:
On the other hand why wouldn't we have it always enabled?

Reply:
The metrics are always exposed from Catalyst API with the `/mistmetrics` endpoint. This is for the endpoint exposed from Mist. And it corresponds to this: https://github.com/livepeer/livepeer-infra/blob/40c743cae8f63421d7344032831dc0a9cb2ea01e/values/catalyst.values.yaml#L6